CMakeLists.txt

```diff
@@ -49,6 +49,17 @@ endif()
 # tests requires hardware.
 option(BUILD_DEV_TESTS "Build dev tests" OFF)
 
+# Already include CTest here to allow it to find tests defined in subdirectories like lib and modules
+if(EVEREST_CORE_BUILD_TESTING)
+    include(CTest)
+    enable_testing()
```

Comment:
```suggestion
```
We can drop this line, since we require BUILD_TESTING=ON

https://cmake.org/cmake/help/latest/command/enable_testing.html